staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/typeconverter.go

```diff
@@ -37,6 +37,6 @@ func NewDeducedTypeConverter() TypeConverter {
 // NewTypeConverter builds a TypeConverter from a proto.Models. This
 // will automatically find the proper version of the object, and the
 // corresponding schema information.
-func NewTypeConverter(openapiSpec *spec.Swagger, preserveUnknownFields bool) (TypeConverter, error) {
+func NewTypeConverter(openapiSpec map[string]*spec.Schema, preserveUnknownFields bool) (TypeConverter, error) {
```

Comment:
nit: The comment looks to be inaccurate now: `NewTypeConverter builds a TypeConverter from a proto.Models`

Reply:
added more detail. PTAL